tunder/lib/src/console/commands/migrations/migrate_rollback_command.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:tunder/database.dart';
+import 'package:tunder/src/console/commands/migrations/contracts/migration_command.dart';
+import 'package:tunder/src/console/commands/migrations/mixins/manage_migrations.dart';
+
+class MigrateRollbackCommand extends MigrationCommand with ManageMigrations {
+  final name = 'migrate:rollback';
+  final description = 'Rollback the last database migration';
+  final List<Migration> migrations;
+
+  MigrateRollbackCommand(this.migrations) {
+    this.migrations.sort((a, b) => a.id.compareTo(b.id));
+  }
+
+  Future<int> run() async {
+    final ranMigrations = await getRanMigrations() as List<MappedRow>;
+    if (ranMigrations.isEmpty) {
+      info('No migrations to rollback');
+      return 0;
+    }
+
+    final migrationId = ranMigrations.last['id'] as String;
+    final migration = migrations.where((mig) => mig.id == migrationId).first;
+    final rollingback =
+        progress('Rolling back: ${migration.id}_${migration.name.snakeCase}');
+
+    try {
+      await migration.down();
```

Comment:
The up() and down() methods could have a more meaningful name. Unless they're common words in Postgres or Laravel. A few suggestions are undo, further, perform, advance and execute. If none of these words are more precise about the operation that is being performed by these methods, then just dismiss this suggestion.

Reply:
The methods `up` and `down` are very well known for migrations in different frameworks like Laravel, rails, and knex.

Reply:
Thanks for letting me know, they're the correct term indeed.